tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -282,20 +311,20 @@ TEST(tfw_str_eq_cstr, handles_empty_strs)
 {
 	TfwStr s1 = {
 		.len = 0,
-		.ptr = (void *)"garbage"
+		.data = (char *)"garbage"
 	};
 	TfwStr s2 = {
 		.len = 0,
-		.ptr = NULL
+		.data = NULL
 	};
 	TfwStr chunks[] = { s1, s2 };
 	TfwStr s3 = {
 		.len = 0,
-		.ptr = &chunks
+		.chunks = (struct TfwStr *)&chunks
 	};
 	TfwStr s_ne = {
 		.len = 3,
-		.ptr = (void *)"foo"
+		.data = (void *)"foo"
```

Comment:
There are macros `DEFINE_TFW_STR()` and  `TFW_STR_FROM()` that can be used for this purpose.